extensions/amp-story-shopping/0.1/amp-story-shopping.js

```diff
@@ -4,6 +4,35 @@ import {AmpStoryShoppingTag} from './amp-story-shopping-tag';
 
 import {CSS as shoppingCSS} from '../../../build/amp-story-shopping-0.1.css';
 
+/**
+ * Language and currency sensitive number formatting.
+ * @param {../../../src/service/localization.LocalizationService} LocalizationService
+ * @param {Element} el
+ * @param {string} currency // An ISO 4217 currency code.
+ * @param {number} price
+ * @return {string}
+ */
+export const formatI18nNumber = (LocalizationService, el, currency, price) => {
+  const langCode = LocalizationService.getLanguageCodesForElement(el)[0];
+  return new Intl.NumberFormat(langCode, {style: 'currency', currency}).format(
+    price
+  );
+};
+
+/**
+ * @param {!Window} win
+ * @param {!Array<!Object>} fontFaces with urls from https://fonts.googleapis.com/css2?family=Poppins
```

Comment:
You can remove the "from https://fonts.googleapis.com/css2?family=Poppins" part because this method should be able to load any given font